src/components/global/TextInput.jsx

```diff
@@ -0,0 +1,16 @@
+function TextInput(props) {
+  const { label, textValue, placeholderValue, onChangeHandler } = props;
+  
+  return (
+    <div className="text-input">
+      {label && 
+        <label className="text-input--label">
```

Comment:
we've been using '--' in class names for conditions/states, for example button--disabled. Maybe consider changing this to a single -